plugins/inputs/lustre2/lustre2_test.go

```diff
@@ -330,3 +333,38 @@ func TestLustre2GeneratesJobstatsMetrics(t *testing.T) {
 	err = os.RemoveAll(os.TempDir() + "/telegraf")
 	require.NoError(t, err)
 }
+
+func TestLustre2CanParseConfiguration(t *testing.T) {
+	config := []byte(`
+[[inputs.lustre2]]
+   ost_procfiles = [
+     "/proc/fs/lustre/obdfilter/*/stats",
+     "/proc/fs/lustre/osd-ldiskfs/*/stats",
+   ]
+   mds_procfiles = [
+     "/proc/fs/lustre/mdt/*/md_stats",
+   ]`)
+
+	table, err := toml.Parse([]byte(config))
+	assert.NoError(t, err)
```

Comment:
Use `require.NoError(`, since this will stop the test if it fails and other follow on errors which will just be confusing.  I almost always use `require` instead of `assert`.

Reply:
Happy to 👍 My (rough) rule of thumb has always been `require` when I need to fail fast and its not something I am directly testing. i.e. my test is focussed on behavior A, but there are some error conditions which need to be nil before I can assert that. Saying that, I totally disregarded that rule here 😂

Reply:
I can see that from a language point of view, require the pre-conditions and then make the assertion.  But the semantic difference between the two forms is "continue or not", and I've found the answer is virtually always no.  In practice its probably moot since a good test checks one thing, and the assert would just be the last check.